doc/Learn/Tutorials/Markup/HowTo-CustomMarkupProject-Toolkit.md

```diff
@@ -0,0 +1,540 @@
+				    new Button()
+					    .Content("Go to Main Page")
+					    .Assign(out var navigationButton)
+			    ));
+
+		    navigationButton.Click += (s, e) =>
+		    {
+			    Frame.Navigate(typeof(MainPage));
+		    };
+	    }
+    }
+
+    ```
+
+    Now we can change the content of the MainPage.cs to have the NavigationBar.
+
+    ```csharp
+    new NavigationBar().Content("Title Main Page")
+    ```
+
+    And For have the controls to navigate for the SecondPage we can add a Button for do it.
+
+    ```csharp
+    new Button()
+		.Content("Go to Second Page")
+		.Assign(out var navigationButton),
+    ```
+
+    The main ideia here is to create a button that have Assign it self the output variable named in this case as navigationButton.
+    After that we need to add a Click EventHandler to add the Navigation using the Frame.Navigate. 
+
+    > Notice how simple it is to create an action for the Button's Click event.
+
+    ```csharp
+    
+		navigationButton.Click += (s, e) =>
+		{
+			Frame.Navigate(typeof(SecondPage));
+		};
+    ```
+
+    # [**XAML**](#tab/cli)
+    
+    #### XAML
+
+    To do so, create a new Page file named SecondPage.
+    Then open the Page.xaml and replace the content for:
+
+    ```xml
+    <Page x:Class="MySampleToolkitProjectXAML.SecondPage"
+	    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	    xmlns:local="using:MySampleToolkitProjectXAML"
+	    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	    mc:Ignorable="d"
+	    xmlns:utu="using:Uno.Toolkit.UI"
+	    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+
+	    <StackPanel HorizontalAlignment="Center"
+				    VerticalAlignment="Center">
+		    <utu:NavigationBar Content="Title Main Page"/>
+		    <Button x:Name="navigationButton" Content="Go to Second Page" Click="navigationButton_Click"/>
+	    </StackPanel>
+    </Page>
+    ```
+
+    And the SecondPage.cs
+
+    ```csharp
+    namespace MySampleToolkitProjectXAML;
+
+    public sealed partial class SecondPage : Page
+    {
+	    public SecondPage()
+	    {
+		    this.InitializeComponent();
+	    }
+	    public void navigationButton_Click(object sender, RoutedEventArgs e)
+	    {
+		    Frame.Navigate(typeof(MainPage));
+	    }
+    }
+    ```
+
+    Now we can change the content of the MainPage.xaml to have the NavigationBar.
+
+    ```xaml
+    <utu:NavigationBar Content="Title Main Page"/>
+    ```
+
+    And For have the controls to navigate for the SecondPage we can add a Button for do it.
+
+    ```csharp
+    <Button 
+			Content="Go to Second Page" 
+			Click="navigationButton_Click"/>
+    ```
+
+    # [**Full Code**](#tab/code)
+
+    #### Full C# Markup code
+    
+    - Example of the complete code on the MainPage.cs, so you can follow along in your own project.
```

Comment:
```suggestion
    Now we can change the content of the MainPage.xaml to have the NavigationBar.

    ```xaml
    <utu:NavigationBar Content="Title Main Page"/>
    ```

    And to have the controls to navigate for the SecondPage we can add a Button to do it.

    ```csharp
    <Button 
			Content="Go to Second Page" 
			Click="navigationButton_Click"/>
    ```

    # [**Full Code**](#tab/code)

    #### Full C# Markup code
    
    - Example of the complete code on the MainPage.cs, so you can follow along in your own project.
```